extensions/youtrack/src/work-item.tsx

```diff
@@ -0,0 +1,88 @@
+import { IssueExtended, WorkItemSubmit } from "./interfaces";
+import { WorkItem } from "youtrack-rest-client";
+import { useEffect, useState } from "react";
+import { Action, ActionPanel, Detail, Form, Icon, showToast, Toast, useNavigation } from "@raycast/api";
+
+export function AddWork(props: {
+  getIssueDetailsCb: () => Promise<IssueExtended> | null;
+  createWorkItemCb: (workItem: WorkItem) => Promise<WorkItem> | null;
+  link: string;
+  instance: string;
+}) {
+  const [issue, setIssue] = useState<IssueExtended | null>(null);
+  const [submitted, setSubmitted] = useState(false);
+  const { pop } = useNavigation();
+  useEffect(() => {
+    async function fetchIssueDetails() {
+      const issue = await props.getIssueDetailsCb();
+      setIssue(issue);
+    }
+
+    fetchIssueDetails();
+  }, []);
+
+  if (!issue) {
```

Comment:
Will this ever be false if there is an error?

Reply:
I'm not sure if I understand your intention here. My reason for this condition is to show the loading indicator in early render processes when the `issue` is `null`.